core/store/src/trie/state_parts.rs

```diff
@@ -503,6 +497,16 @@ impl Trie {
             |err| StorageError::StorageInconsistentState(format!("Failed to decode node: {err}")),
         )
     }
+
+    fn retrieve_node_internal(
```

Comment:
There is also `internal_retrieve_trie_node`. Can we consider better naming for these both?

Reply:
Hmm right... I can change this name to `retrieve_storage_node`